### PR TITLE
Adding support to get payload parameters and to get the deploy task

### PR DESCRIPTION
**Pull request by heim**
If one uses `hubot-deploy` you can send arbitrary fields as a payload, this PR enables you to pull the payload fields by using `deployment-get-payload <field_name>`. Does not work for nested fields.

PR also adds the ability to get the deploy task as sent by `hubot-deploy`.

### CODE REVIEW

**Comment by heim** [comment]
PR has not been tested, so hold off merging.

**Comment by heim** [comment]
@judoole Do you think `deployment-get-payload` should do a `exit 1` if not given an argument (or too many arguments)? Right now it just fails silently.

**Comment by judoole** [comment]
Yes, I think `exit 1` sounds smart! Or `exit 64` https://www.freebsd.org/cgi/man.cgi?query=sysexits

> PR has not been tested, so hold off merging.

Maybe we should take a look at that hanging issue bats #10

**Comment by heim** [comment]
> Yes, I think `exit 1` sounds smart! Or `exit 64` https://www.freebsd.org/cgi/man.cgi?query=sysexits

`exit 64` is probably the best one, yes.
 
> > PR has not been tested, so hold off merging.
> 
> Maybe we should take a look at that hanging issue bats #10

Agreed.

**Review by judoole — APPROVED** [review]

**Pull request merged**
